setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.2.1
+current_version = 0.2.2
```

Comment:
`major.minor.patch`

I would assume this is a breaking change.
I suggest to increase the minor number for breaking changes